br/pkg/lightning/restore/table_restore.go

```diff
@@ -710,8 +710,8 @@ func (tr *TableRestore) postProcess(
 	}
 
 	// tidb backend don't need checksum & analyze
-	if !rc.backend.ShouldPostProcess() {
-		tr.logger.Debug("skip checksum & analyze, not supported by this backend")
+	if rc.cfg.PostRestore.Checksum == config.OpLevelOff && rc.cfg.PostRestore.Analyze == config.OpLevelOff {
+		tr.logger.Debug("skip checksum & analyze, either because not supported by this backend or manually disabled")
```

Comment:
are these (f52820108600c86757531fc3b182fb8012a00b9a) part of this PR?

Reply:
No. As this is just a tiny change, maybe it not worth to file another PR(as while as an issue)😅。